examples/analysis/analyzer-results.txt

```diff
@@ -1,5 +1,7 @@
 Analyzing analysis_options.yaml, lib...
   error • A value of type 'Object' can't be assigned to a variable of type 'String'. • lib/assignment.dart:11:14 • invalid_assignment
+  lint • Omit type annotations for local variables. • lib/assignment.dart:10:3 • omit_local_variable_types
+  lint • Omit type annotations for local variables. • lib/assignment.dart:12:3 • omit_local_variable_types
```

Comment:
Consider ignoring `omit_local_variable_types` (since that error isn't one that is being illustrated by that code), maybe around this line: https://github.com/dart-lang/site-www/blob/8fcfdd7536853a265378d1a515637986f8659c64/examples/analysis/lib/assignment.dart#L4

Reply:
Yup, I was just doing that. :) I tried that the first time and it didn't seem to work, for some reason.

Reply:
You got this! 👍

Reply:
For some reason I'm getting an error when I make that change:

Failed to precompile build_runner:graph_inspector:

Reply:
I probably specified it wrong. I'll play with the analysis options ignores

Reply:
Please see https://github.com/dart-lang/sdk/issues/39702

Reply:
Yep, that's the problem. Thanks!